pwa/src/templates/organizationDetail/OrganizationDetailTemplate.tsx

```diff
@@ -31,158 +33,174 @@ interface OrganizationDetailTemplateProps {
+                  )}
+                  {_getOrganization.data.gitlab && (
+                    <ToolTip tooltip="GitLab">
+                      <Tag
+                        label={t("GitLab")}
+                        icon={<GitLabLogo />}
+                        onClick={() => open(_getOrganization.data.gitlab)}
+                      />
+                    </ToolTip>
+                  )}
+                  {_getOrganization.data.website && (
+                    <ToolTip tooltip={"Website"}>
+                      <Tag
+                        label={_getOrganization.data.website}
+                        icon={<FontAwesomeIcon icon={faGlobe} />}
+                        onClick={() => open(_getOrganization.data.website)}
+                      />
+                    </ToolTip>
+                  )}
+                  {_getOrganization.data.phone && (
+                    <ToolTip tooltip={"Telefoonnummer"}>
+                      <Tag
+                        label={_getOrganization.data.phone}
+                        icon={<FontAwesomeIcon icon={faPhone} />}
+                        onClick={() => navigate(`tel:${_getOrganization.data.phone}`)}
+                      />
+                    </ToolTip>
+                  )}
+                  {_getOrganization.data.email && (
+                    <ToolTip tooltip={"EmailAddress"}>
+                      <Tag
+                        label={_getOrganization.data.email}
+                        icon={<FontAwesomeIcon icon={faEnvelope} />}
+                        onClick={() => navigate(`mailto:${_getOrganization.data.email}`)}
+                      />
+                    </ToolTip>
+                  )}
+                </div>
+              </div>
+
+              <Divider />
+
+              <div className={styles.tagsContainer}>
+                <ToolTip tooltip={"ISO-9001"}>
                   <Tag
-                    label={tempOrganization.phone}
-                    icon={<FontAwesomeIcon icon={faPhone} />}
-                    onClick={() => navigate(`tel:${tempOrganization.phone}`)}
+                    label={"ISO-9001"}
+                    icon={<FontAwesomeIcon icon={faCertificate} />}
+                    onClick={() => open("https://www.iso.org/iso-9001-quality-management.html")}
                   />
                 </ToolTip>
-              )}
-              {tempOrganization.email && (
-                <ToolTip tooltip={"EmailAddress"}>
+                <ToolTip tooltip={"ISO-27001"}>
                   <Tag
-                    label={tempOrganization.email}
-                    icon={<FontAwesomeIcon icon={faEnvelope} />}
-                    onClick={() => navigate(`mailto:${tempOrganization.email}`)}
+                    label={"ISO-27001"}
+                    icon={<FontAwesomeIcon icon={faCertificate} />}
+                    onClick={() => open("https://www.iso.org/isoiec-27001-information-security.html")}
                   />
```

Comment:
Are the licenses not coming back with the API yet? If not, could we create a issue for this?